### PR TITLE
PLT-5267 Relocated obsolete examples 

### PR DESCRIPTION
**Pull request by bwbush**
Obsolete examples and tutorials are moved to https://github.com/input-output-hk/real-world-marlowe/tree/main/archives. The commit history was preserved during this migration.

Pre-submit checklist:
- Branch
    - [ ] Tests are provided (if possible)
    - [ ] [Test report is updated](https://github.com/input-output-hk/marlowe-cardano/blob/main/marlowe/test/test-report.md) (if relevant)
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [x] Relevant tickets are mentioned in commit messages
    - [x] Formatting, PNG optimization, etc. are updated
- PR
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
        - Review required
        - [x] Substantial changes to code, test, or documentation
        - [ ] Change made to Marlowe validator (@bwbush and @palas must be included as reviewers)
        - Review not required
        - [ ] Minor changes to non-critical code, documentation, nix derivations, configuration files, or scripts
        - [ ] Formatting, spelling, grammar, or reorganization
    - [x] Reviewer requested

### CODE REVIEW

**Review by ladamesny — APPROVED** [review]
Looks good - must feel great to get rid of so many files. There is one change needed before we can merge. But good to go after that change.

File: marlowe-cli/ReadMe.md
Comment (bwbush) [reply marlowe-cli/ReadMe.md]: Thanks!

**Pull request merged**
